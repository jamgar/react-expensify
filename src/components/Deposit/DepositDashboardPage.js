import React from 'react'
import DepositsSummary from './DepositsSummary'
import DepositList from './DepositList'
import DepositListFilters from './DepositListFilters'

const DepositDashboardPage = () => (
  <div>
    <DepositsSummary />
    <DepositListFilters />
    <DepositList />
  </div>
)

export default DepositDashboardPage
