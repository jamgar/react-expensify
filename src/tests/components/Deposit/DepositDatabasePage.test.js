import React from 'react'
import { shallow } from 'enzyme'
import DepositDashboardPage from '../../../components/Deposit/DepositDashboardPage'

test('should render DepositDashboardPage correctly', () => {
  const wrapper = shallow(<DepositDashboardPage />)
  expect(wrapper).toMatchSnapshot()
})
