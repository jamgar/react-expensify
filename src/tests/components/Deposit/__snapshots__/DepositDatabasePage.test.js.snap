// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DepositDashboardPage correctly 1`] = `
<div>
  <Connect(DepositsSummary) />
  <Connect(DepositListFilters) />
  <Connect(DepositList) />
</div>
`;
